app/event_source/subscribers/private_family_member_subscriber.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Subscribers
+  # Subscriber will receive private person saved requests
+  class PrivateFamilyMemberSubscriber
```

Comment:
@jacobkagon We do not need another subscriber. We should be able to use the previous subscriber. Just rename the class to be more generic instead of PrivatePersonSubscriber and add a new block to handle family_member_created event.